src/runtime/virtcontainers/pkg/cgroups/utils.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2020 Intel Corporation
```

Comment:
Not entirely sure -- does this need updating?

Reply:
I'll let Intel decide about it?